python/pyproject.toml

```diff
@@ -48,6 +51,8 @@ zstandard = { version = "^0.18.0", optional = true }
 
 python-snappy = { version = "^0.6.1", optional = true }
 
+thrift = "^0.16.0"
```

Comment:
Can this be part of an optional `hms` or `hive` dependency bundle? I don't think that we need to install it by default.

Reply:
Good call, I've made it optional, and you can install it using an extra: `pip3 install 'pyiceberg[hive]'`